config/parse-overrides/src/index.ts

```diff
@@ -41,10 +41,19 @@ export function parseOverrides (
 function parsePkgSelector (selector: string) {
   const wantedDep = parseWantedDependency(selector)
   if (!wantedDep.alias) {
-    throw new PnpmError('INVALID_OVERRIDE_SELECTOR', `Cannot parse the "${selector}" selector in the overrides`)
+    throw new ParseSelectorError('INVALID_SELECTOR', selector)
   }
   return {
     name: wantedDep.alias,
     pref: wantedDep.pref,
   }
 }
+
+class ParseSelectorError extends PnpmError {
```

Comment:
I can’t do this until after work, but realize with how this is structured now, this class doesn’t need to exist. I’ll remove this and just throw a PnpmError here